examples/all-clusters-app/esp32/README.md

```diff
@@ -18,28 +18,30 @@ CHIP.
 
 ## Supported Devices
 
-The CHIP demo application is intended to work on two categories of ESP32
-devices: the
+The CHIP demo application is intended to work on two categories of ESP32 devices
+and one categories ESP32C3 device: the
 [ESP32-DevKitC](https://www.espressif.com/en/products/hardware/esp32-devkitc/overview),
-and the [M5Stack](http://m5stack.com). On the [M5Stack](http://m5stack.com) this
-example displays a CHIP QRCode with the device's Soft-AP SSID encoded in the TLV
-section.
+the [M5Stack](http://m5stack.com) and the
+[ESP32C3-DevKitM](https://docs.espressif.com/projects/esp-idf/en/latest/esp32c3/hw-reference/esp32c3/user-guide-devkitm-1.html).
+On the [M5Stack](http://m5stack.com) this example displays a CHIP QRCode with
+the device's Soft-AP SSID encoded in the TLV section.
 
 ## Building the Example Application
 
 Building the example application requires the use of the Espressif ESP32 IoT
-Development Framework and the xtensa-esp32-elf toolchain.
+Development Framework and the xtensa-esp32-elf toolchain(for ESP32C3,the
```

Comment:
the xtensa-esp32-elf toolchain for ESP32, and the riscv-esp32-elf toolchain for ESP32-C3.